### PR TITLE
Changed file formatting

### PR DESCRIPTION
**Pull request by campbelle1**
As per Jesse and Rahim's suggestions, I fixed the formatting of my chapter folder so that an intro page could be built from the data-visualization.md file. Please let me know if I did this correctly.

I ran into a few problems after trying to fix this. (1), I get a warning sign about my use of the 'seaboard-whitegrid' style and (2) My code for importing the data file creates an error. Rahim suggested using a url to resolve issue (2), but for some reason, when I test this on JupyterHub, I get an error. I also saw this error when I built the chapter, but for some reason in my most recent build, it seemed to work. There's some sort of discrepancy between what I see on JupyterHub and what I see on the html build that I don't fully understand....

### CODE REVIEW

**Comment by jesteria** [comment]
Hello again! So I'm working through what you've submitted. But to begin to discuss….

Yes the intro file looks good to me. But I'm a bit confused by what I see in the notebooks.

* Notebooks don't generally carry over variables between them – they're all executed separately. Yet you define a DataFrame, `military`, in 02.ipynb and then expect it to be available to 03.ipynb. …Does this work under any environment?
* That's right, you can't say – `pd.read_csv('NorthAmerica_Military_USD-PercentGDP_Combined.csv')` – when there's no such CSV file in the directory of the notebook. The path has to refer to where to find the file on the filesystem.
  * (More than that, it's problematic to refer to external data files in the notebooks used for this textbook, because the textbook allows readers to download the notebook … but they won't necessarily have other arbitrary files.)
* Yes, you get a DeprecationWarning, which says that `plt.style.use('seaborn-whitegrid')` is no longer the recommended API. The warning is saying: don't do that.
* Your sections are numbered 2-4 – is there a forthcoming section 1?

Anyway, I can fix things up as makes sense to me. But let me know if I'm missing anything.

**Comment by campbelle1** [comment]
Hi Jesse,

Thanks for your response.

1. Yes, the way that I organized the chapter was a continuation of use of the same data divided into separate sections. For the purpose of a student or someone looking at the chapter, I assume that they are reading through the chapter and running code in one session as they are chronologically going through each section. I felt like importing libraries and redefining variables at the beginning of each section would be tedious and reduced readability of the chapter. If that **needs** to be done, I guess I would just I have to compress the whole chapter into one section.

2. Would the working directory that I would reference from be the same working directory that I am in when I am making edits and pushing to Github? I believe I usually work in the /textbook directory. Would it be better to use a url for downloading the file, like Rahim suggested?

3. Does the warning show up on your end? On my end, I still see it, even though I put in the code to ignore the warnings. When I edited the notebook on JupyterHub, I tried using "seaborn-v0_8" as the style to rectify the warning, but I get an error, so it's hard for me to know what will work in the build when I'm getting two different results...

4. There will be no 1 folder, just 2 - 4.

**Comment by jesteria** [comment]
> I felt like importing libraries and redefining variables at the beginning of each section would be tedious and reduced readability of the chapter.

It's easy enough to hide cells from the rendered textbook. The notebook just has to be executable.

> Would the working directory that I would reference from be the same working directory that I am in when I am making edits and pushing to Github?

It's up to making the notebook executable; and, I believe the path is relative to the notebook.

For anything like this, the data can just be embedded in the notebook.

> Does the warning show up on your end?

Yes. I believe the solution is just to use seaborn: `seaborn.set_style('whitegrid')`

But I can take care of this.

**Comment by campbelle1** [comment]
> > It's easy enough to hide cells from the rendered textbook. The notebook just has to be executable.
>
>  Oh okay, I wasn't aware of that. Thanks for letting me know. I guess we could add the second cell from 02.pynb to the top of 03.pynb and 04.pynb and hide them in the rendered textbook.
>
>
>  > It's up to making the notebook executable; and, I believe the path is relative to the notebook. 
>  > For anything like this, the data can just be embedded in the notebook.
> 
> Okay, thanks for clarifying!
> 
> > Yes. I believe the solution is just to use seaborn: `seaborn.set_style('whitegrid')`. 
> > But I can take care of this.
> 
> Okay, so I just checked in 02.pynb when I replaced the line 
> `plt.style.use('seaborn-whitegrid')` 
> with 
> `import seaborn as sns`
> `sns.set_style('whitegrid')`
> it effectively allows for the same plots without me having to change the code for the plots, so if you could make that change in 02.pynb that would be great. Thank you very much for your time, and please let me know if there's anything I need to do.

**Review by jesteria — APPROVED** [review]
OK, I've went through, cleaned up and incorporated everything.

This looks good to me now. But I'll hold off on merging until others have had a look.

Please feel free to review what I've done, and let's discuss, as you like.

And thanks!

File: textbook/08/1/Libraries.ipynb [hunk 1]
```diff
@@ -0,0 +1,401 @@
+{
```
Comment (jesteria) [inline textbook/08/1/Libraries.ipynb:1]: <p>Note: This cell is hidden (collapsed) in the rendered HTML thanks to the cell tag <code>hide_cell</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (jesteria) [reply textbook/08/1/Libraries.ipynb:1]: Well OK that's how ReviewNB works.

Anyway, yes, all those verbose cells are hidden by default.

The problem is that the Notebooks are meant to be downloadable and interacted with by readers; but, there's not yet any real handling for separate data files. There are various ways of handling this; for example, we can load files from a/the server via URL. But, it's useful for the whole process to keep things all together. (It's also not such a bad thing for readers to see the plain text of the data they're interacting with.)

image

**Comment by jesteria** [comment]
OK, this has been merged via 27111e10d2fcd92b49d1211c9aae7ff2f3a73a6c and 54b2aa401fc10cd068448c8786d82588b1dca60e.

(I'd intended to let GitHub handle/see the merge … but then I didn't.)

**Pull request closed without merging**